src/sgx/CMakeLists.txt

```diff
@@ -42,41 +41,41 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 set(SGX_SDK_ENCLAVE_SIGNER ${SGX_SDK_PATH}/bin/x64/sgx_sign)
 set(SGX_SDK_ENCLAVE_EDGER8R ${SGX_SDK_PATH}/bin/x64/sgx_edger8r)
 
-# WAMR build configuration
-set(WAMR_BUILD_TARGET X86_64)
+# --------------------------------------------------------
+# WAMR Build
+# --------------------------------------------------------
+
+# Set target platform details
+set (WAMR_BUILD_PLATFORM "linux-sgx")
+set (WAMR_BUILD_TARGET X86_64)
 
-if(FAASM_SGX_WAMR_AOT_MODE)
+# Set WAMR include flags
+# 28/06/2021 - (fast) interpreter, AOT, libc, wasi, and pthread are enabled by
+# default in WAMR's SGX port.
```

Comment:
Is there a link to where this happens in the WAMR cmake files? Could we include a link? Although it's got a date on it, this comment is likely to become outdated and having a link to where it's actually done would be more time-proof.